legacy/full/Dockerfile

```diff
@@ -92,7 +92,7 @@ RUN curl -fsSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.t
         github.com/go-delve/delve/cmd/dlv \
         github.com/golangci/golangci-lint/cmd/golangci-lint && \
     GO111MODULE=on go get -v \
-        golang.org/x/tools/gopls@v0.7.3 && \
+        golang.org/x/tools/gopls@v0.7.5 && \
```

Comment:
@princerachit Two questions:

1. Why are we keeping the old images under `legacy/`?

2. And, is it okay to continue occasionally bump security patches like this? (If we're keeping these old images, we might as well make them safe to use by cheaply accepting weekly PRs like this 💭)

Reply:
1. There are two reasons - (a) We want to keep a reference so that we can fix if something is missing in new images. (b) Users who are using the legacy tagged image can refer these dockerfiles to understand how those images were built. As we gain confidence with new images, we will get rid of legacy dir altogether.
2. No. We are not supporting old images and would not be releasing anything from legacy dir. It is only for reference and nothing more.

That being said I am going to close this PR.

Reply:
Thanks! Hmm, seems a shame to purposefully let the legacy images bitrot. But maybe this requires a `.autofixignore` file or similar. Will think about it.